docs/releases/connect-release.md

```diff
@@ -1,8 +1,9 @@
 ## @trezor/connect trezor.io release
-
 -   [x] From develop branch create a pull requests into branch `release/connect-v9`
+
+### After merging this PR
+-   [release/connect-v9 pipeline](https://gitlab.com/satoshilabs/trezor/trezor-suite/-/pipelines?page=1&scope=branches&ref=release%2Fconnect-v9)
 -   [ ] Tested and approved by @trezor/qa. Typically using [this build](https://suite.corp.sldev.cz/connect/release/connect-v9/)
```

Comment:
```suggestion
-   [ ] Tested and approved by @trezor/qa. Typically using [this build](https://suite.corp.sldev.cz/connect/release/connect-v9/) Changelog [here](https://github.com/trezor/trezor-suite/blob/develop/packages/connect/CHANGELOG.md).
```

Reply:
link should be https://github.com/trezor/trezor-suite/blob/release/connect-v9/packages/connect/CHANGELOG.md

Reply:
ahhh, right